### ISSUE TITLE
You can buy a house with shop point, from another player has bought already in game

### ISSUE THREAD

**Issue by kungen32** [Znote/ZnoteAAC#463]
Well i have turned around all files from znote and data, i didnt find the solution, the only one was disable buy houses with shop points, because if anyone buy it with shop points, in game don't appears like a house bought, obviously, so you can buy it in  game and the player who has bought house with shop points, lose house.

The znote is fresh from 2 Days ago, data is Custom tfs 1.3 , using same system as otservbr 12.

**Comment by Znote** [comment]
Disable it in-game and only use web for house auction and direct purchase?

**Comment by Znote** [comment]
Using in-game house system together with web house system is bad, because ingame house systems usually ignores bidding round and goes straight into giving ownership. There is also a chance the house table is cached, so web won't know about any changes to the table (like new owners) until server save. (Not quite sure about this though).

**Comment by kungen32** [comment]
Yes, like a ' instant see ', like hischores, is not instantly , i got this after i understand lol.

I have disable in web the auctions with shop, is working correctly.

The only last thing on znote is working +/- is payment with paypal, some players got points other no, but i see your post about this saying anything about update of paypal, for example 100€ anyone recived points, but for 20, no.

Thanks for the tip.
